app/src/main/java/ar/com/wolox/android/example/ui/home/fragment/HomeFragment.kt

```diff
@@ -0,0 +1,51 @@
+package ar.com.wolox.android.example.ui.home.fragment
+
+import androidx.viewpager2.widget.ViewPager2
+import ar.com.wolox.android.R
+import ar.com.wolox.android.databinding.FragmentHomeBinding
+import ar.com.wolox.android.example.ui.home.ViewPagerAdapter
+import ar.com.wolox.android.example.ui.home.news.NewsFragment
+import ar.com.wolox.android.example.ui.home.userProfile.UserProfileFragment
+import ar.com.wolox.wolmo.core.fragment.WolmoFragment
+import com.google.android.material.tabs.TabLayout
+import com.google.android.material.tabs.TabLayoutMediator
+import javax.inject.Inject
+
+class HomeFragment private constructor() : WolmoFragment<FragmentHomeBinding, HomePresenter>(), HomeView {
+
+    private lateinit var homeAdapter: ViewPagerAdapter
+    private lateinit var pager: ViewPager2
+
+    @Inject
+    internal lateinit var newsFragment: NewsFragment
```

Comment:
I think you don't need to make this and the UserProfileFragment `internal`

Reply:
In the boostrap example, ViewPagerFragment, both randomFragment and requestFragment are `internal` . For consistency i left both of my fragments internal.